src/precice/impl/DataContext.hpp

```diff
@@ -79,7 +79,7 @@ class DataContext {
    * @param[in] mappingContext Context of the mapping
    * @param[in] meshContext Context of mesh this mapping is mapping from or to
    */
-  virtual void configureMapping(const MappingContext &mappingContext, const MeshContext &meshContext) = 0;
+  virtual void addMappingConfiguration(const MappingContext &mappingContext, const MeshContext &meshContext) = 0;
```

Comment:
Please also update the class documentation above that a data context can have no, one or multiple mappings.

Reply:
I don't have a strong opinion on this, so I just changed to the append version.